py3status/modules/moc.py

```diff
@@ -132,15 +138,14 @@ def moc(self):
         return {
             'cached_until': self.py3.time_in(cached_until),
             'color': color,
-            'full_text': self.py3.safe_format(self.format,
-                                              dict(
-                                                  is_paused=is_paused,
-                                                  is_playing=is_playing,
-                                                  is_started=is_started,
-                                                  is_stopped=is_stopped,
-                                                  **data
-                                              ))
-        }
+            'full_text': self.py3.safe_format(
+                self.format, dict(
+                    is_paused=is_paused,
+                    is_playing=is_playing,
+                    is_started=is_started,
+                    is_stopped=is_stopped,
+                    **data
+                ))}
```

Comment:
I really hate these `))}` type things I much prefer
```
(
    (
         {

         }
    )
)
```

to
```
(
    (
         {

         }))
```

because to me it is much easier to read.  Different strokes and all that but like @ultrabug and his alphabetical parameters for me this is very important.

Also I'd do 
```
self.py3.safe_format(
    self.format,
    dict(
        ...
``` as again for me, far easier to read

Reply:
Madness. All of this introduces more lines and feels clunky to read. I agree on this.
```
return {
     'simple': 'simple',
     'thing': 'thing'
}
```
using `''` instead of `dict()` and to have its own `}`. I think the problem is using `dict()` itself... and if we do the example above, it'll break Python2 as it reported `No Import` something when I do `**data` inside the example.

Reply:
well `}))` is a blocker for me just so you know

Reply:
Why not? It's your birthday today. 🎂 🎁 🎈